tools/issue-labeler/src/SearchIndexCreator/IssueIndex.cs

```diff
@@ -263,8 +263,14 @@ private SearchIndex GetSampleIndex()
                         IsSearchable = false
                     },
                     new SearchableField("Title"),
-                    new SearchableField("Service"),
-                    new SearchableField("Category"),
+                    new SearchableField("Service")
```

Comment:
At some point, we'll need to find a more generic way to do this.   Or, we'll need to deal with it being super-specific to us and allow for some dynamic plugin system for loading custom labelers/answer thingers on demand.